floss/language_identifier.py

```diff
@@ -149,3 +154,11 @@ def verify_pclntab(section, pclntab_va: int) -> bool:
         logger.error("Error parsing pclntab header")
         return False
     return True if pc_quanum in {1, 2, 4} and pointer_size in {4, 8} else False
+
+
+def is_dotnet_bin(sample: str) -> bool:
+    """
+    The 'net' attribute of a dnPE object is null if the file is not a dotnet binary.
+    """
+    file = dnfile.dnPE(sample)
+    return bool(file.net)
```

Comment:
for reference, this property is populated here:

https://github.com/malwarefrank/dnfile/blob/527d57033720482c22451be448b743f3590ca7d0/src/dnfile/__init__.py#L173-L181